Awful.apk/src/main/java/com/ferg/awfulapp/task/AwfulRequest.java

```diff
@@ -242,7 +243,7 @@ protected VolleyError customizeAlert(VolleyError error){
     /**
      * Whether or not to automatically check for common page errors during the request process.
      * Override it and return false to disable these checks.
-     * @see handleError() and AwfulError.checkPageErrors() for more details.
```

Comment:
This was already broken huh - can you change it to
``     * See {@link #handleError} and {@link AwfulError#checkPageErrors} for more details.``
thanks!